configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
@vadz

Can you commit the changes I have done.
I do NOT have the time right now to figure out how to do the images and relearn how to edit the doc files and setup.h files.

Thank you.

Tim S.